### ISSUE TITLE
Design suggestion about osu! client download page

### ISSUE THREAD

**Issue by zen6502** [ppy/osu-web#10823]
I think osu!lazer download section needs to be more eye-catching and attractive, so it can make new players lean towards downloading lazer instead of the stable client.

I've made a crappy prototype design in GIMP so you get the idea.
![osu! download page](https://github.com/ppy/osu-web/assets/142730179/cdc20839-63a1-4712-9af3-03dc2ebf14f0)

**Comment by cl8n** [comment]
no comment on the aesthetics, but I think the current download page is already good in terms of offering the user choice between stable/lazer and concisely explaining the difference. imo new users should not be steered toward one option or the other, beyond being offered some information to help understand what each version is.

**Comment by peppy** [comment]
> new users should not be steered toward one option or the other,

yet. let's hope that changes in the next couple of months!

**Comment by DanielPower** [comment]
With how far Lazer has come both in terms of features, performance/stability, and player base (nearing 20% adoption based on https://status.ppy.sh), I think this is worth considering again.

While there are definitely still things holding back many existing players from switching over, I feel like for a new player starting the game today, Lazer provides a better experience. The current website design makes stable feel like the default that the majority of people should pick, and lazer like a beta build. Which in my opinion is no longer the best way to frame it for new players.

**Comment by peppy** [comment]
I've also got this tracked at the top end of my next-up list. @ppy/team-web let's get things moving here.

@vatei may be interested in taking a shot at potential design changes while we're here.

**Comment by cl8n** [comment]
> yet. let's hope that changes in the next couple of months!

hello in 2025

yeah. lazer is a much more recommendable default choice now so it wouldn't feel wrong to me to present it "first" or whatever the equivalent is in the new design. and it shouldn't be framed as a beta anymore either. I still think it'd be weird to have anything explicitly suggesting you *shouldn't* use stable though bc it's still what most players know as the normal game, even if it's true that it's not being developed actively anymore

having some images and/or small videos of each would be good too. could help people figure out which version their friend was playing or which version they saw on a livestream, etc

**Comment by vatei** [comment]
Here's a simple proposal, I guess it looks okay? (I don't know how to include everything mentioned above without overloading the screen)

<img width="1900" height="940" alt="Image" src="https://github.com/user-attachments/assets/128e1b24-5b94-430f-8af7-5727ba532473" />

https://github.com/user-attachments/assets/8dbfa74f-bdb9-41a1-a5aa-2195684bbf75

<img width="457" height="940" alt="Image" src="https://github.com/user-attachments/assets/c4e16923-1787-4ce7-ae51-962c7e6c1527" />

https://github.com/user-attachments/assets/6135ce7a-612e-49d4-98ba-47615fce6c04

**Comment by peppy** [comment]
Some of the copy should also be changed. I'm not sure about the black video or the pink outline thing though.

**Comment by vatei** [comment]
<img width="1900" height="935" alt="Image" src="https://github.com/user-attachments/assets/38048928-056a-4097-b6db-d0c79a27e45e" />

Does that feel better?


> Some of the copy should also be changed

Hmm, which part and in what way?

**Comment by diquoks** [comment]
> Does that feel better?

maybe [this map](https://osu.ppy.sh/beatmapsets/1841885) with storyboard will be better?
and gameplay like in 5.04*

**Comment by Joehuu** [comment]
Seems like showing gameplay is the same as the preview in https://osu.ppy.sh/ (logged out). Maybe show other things listed in https://osu.ppy.sh/wiki/en/Help_centre/Upgrading_to_lazer#feature-comparison in visual form or something?

**Comment by tadatomix** [comment]
How about making something like this https://osu.ppy.sh/home/support, but with "Why you should try lazer"? I think it might encourage more people to see what it is about and _maybe_ clear up some doubts people still have

P.S. If it's not important you can't be bothered to read what I'll write bellow
**What I think should be mentioned in this page, if it plans to be implemented:**

1. osu! on almost any device (basically means while stable is normally playable only on Windows PC, lazer can be played even on the phone);
2. _semi_ Skin layout editor (semi, since still not the majority of things can be changed _yet_);
3. Soft deletion;
4. Fun mods;
5. Mod Presets
6. Some mention of osu!web overlays;
7. _maybe_ Multiplayer spectating;
8. Playlists;
9. Daily Challenges;
10. Replay inspector;

Can't think of anything else. In case you have to add, feel free. I genuinely think this would be more user-friendly than current osu!wiki article. This doesn't mean that it should be nuked, but I think there still should be some sort of entry-point for players to understand much better what unique feature lazer has, compared to stable.

**Comment by peppy** [comment]
The download page shouldn't be geared towards converting users IMO. It should present our recommended default and leave it at that.

**Comment by tadatomix** [comment]
If that's related toward me, that's not what I really meant. My idea wasn't forcing people on getting lazer right now due to made up reasons. All of my offerings are just converting the wiki article about lazer or adding an additional page between the download and wiki, which would present some good sides of lazer, and answer people what they can achieve with getting lazer instead of stable.

If it's related to the actual download page, then I'm feeling confused, since all what vatei did was tidying up this initial style made in GIMP. Current one just gives you 2 equally weighted choices, placing stable on the left (majority of people read from left to right) which quite implies that stable first, lazer after. Yet so, I'm not a fan of wording "yeah, stable is now for oldies", with also toned down text. I don't think that stable should be put in the attic and catching on dust yet. So it's better to at least give the same color to the stable's text. Don't think banner's idea is bad, but IMO it shouldn't be the video. Getting the video from the main page is even worse, considering newcomer experience. Person opens the site, sees the video, clicks on download and it's the same video. It would've give the confusion. What I think might be good is maybe some little art there? Or some graphical design thingy? Something not really being an eyesore, but a simple addition to the lazer. 

TL;DR My page just for good sides of lazer and not enforcing people to get it right now. Download design, I think quite tugs up the blanket to lazer, but some better wording and style fixes might fix it IMO

**Comment by diquoks** [comment]
> All of my offerings are just converting the wiki article about lazer or adding an additional page between the download and wiki, which would present some good sides of lazer, and answer people what they can achieve with getting lazer instead of stable.

you can do it yourself if you want to 🙌
https://osu.ppy.sh/wiki/Help_centre/Upgrading_to_lazer

**Comment by arflyte** [comment]
I'm okay with the overall design. If the intention is to recommend lazer as default, yeah, this will work.

Probably just need a little bit more aesthetic adjustments. Like making the banner full bleed and space adjustments for the copy. Also don't mix alignment in the banner; keeping everything left should be fine. Maybe find a better bg the complements the colour of the site.

**Comment by peppy** [comment]
@ppy/team-web this needs priority. @notbakaneko maybe move focus to here ASAP?

**Comment by notbakaneko** [comment]
Are we going to keep the video guide? We'd need an updated one for lazer if we are 🤔

**Comment by Walavouchey** [comment]
is there any interest in updating or adding a new version of https://assets.ppy.sh/media/landing.mp4 for this page? seeing the above design, it might not need to include anything different, but just to spitball:

- show other game modes (since the gameplay designs have been stable for quite a while)
- show other bundled/fa maps (perhaps ones with storyboards or custom-made videos)
- show the game ui (though it may make the video too busy visually)
- include gameplay but design the background to work with the website ui (like flyte mentioned)

**Comment by iopq** [comment]
I have no lazer download at all on the download page

<img width="1408" height="1035" alt="Image" src="https://github.com/user-attachments/assets/40c5e602-2cae-4a06-9d83-70c39473f687" />

**Comment by Joehuu** [comment]
I believe the lazer code name is being phased out to just osu!.

Closing as resolved in https://github.com/ppy/osu-web/pull/12523.
